src/main/scala/nl.knaw.dans.easy.archivebag/package.scala

```diff
@@ -46,4 +46,7 @@ package object archivebag {
         HttpOptions.followRedirects(false),
       ))
   }
+
```

Comment:
These implicit conversions are very neat, but might they not be confusing later on?